spec/controllers/authenticate_controller_spec.rb

```diff
@@ -6,6 +6,45 @@
   let(:password) { "password" }
   let(:login) { "u-#{random_hex}" }
   let(:account) { "rspec" }
+  let(:authenticator) { "authn" }
+
+  context "#login" do
+    shared_examples_for "successful authentication" do
+      it "succeeds" do
+        post :login, params
+        expect(response).to be_ok
+        expect(response.body.length).to be >= 44
```

Comment:
Could we possible seed the response values with some clever use of `allow` and then check the exactly response value rather than a length here? You can pass an expected value into the shared example I believe.

Reply:
Good idea, I'll take a look into that.